script/Arcueid/Day 13/13_00_ARC13_5.txt

```diff
@@ -518,42 +565,43 @@ For that reason, his eyes reflect only fear.
 [sha:ddd9fd929ddc23858609e9b3a3dc31c1073812e4]{
 -- Page 437, Offset 32922. Glued.
 -- 炎になったようだ。
- is like a blaze.
+ feels like it's on fire.
 }
 [sha:cc6c897b59d0dc8cfd4b493d5ffc93dfbced1577]{
 -- Page 437, Offset 32923. Mods: @k, @e.
 -- 「おまえは命を視ているだけだ。死を理解していない。
-"All you see is life. Death is something you cannot understand.
+"All you're seeing is life. You don't understand death at all.
 }
 [sha:4ef0013a8c7438a777e3ea082b6e74e3d6e49ad8]{
 -- Page 437, Offset 32924.
 -- だから俺も殺せず、弱りきった女しか殺せない」
-That's why you cannot kill me, or even a frail woman."
+That's why you could barely kill a weakened woman.
+ You have no hope of killing me."
 }
 [sha:6223adf21bb80d1d4fbcb81fab7b1469fe7ff9ca]{
 -- Page 437, Offset 32925.
 -- ―――脳髄が、白熱する。
-―――My brain turns incandescent.
+―――My brain glows white hot.
 }
 [sha:df5fa14a2a2c7c1101496292d6ece93d522118ae]{
 -- Page 438, Offset 32926. Mods: @n.
 -- 「死が視えているのなら、
-"If you can really see death,
+"If you could really see death,
 }
 [sha:e2b7105d3f1c2394b93cfb15008b828484ce7e57]{
 -- Page 438, Offset 32927. Glued.
 -- 正気でなんかいられない。おまえに分かるのは物を生かしている部分だけだ。
- then you wouldn't be able to stay sane. What you understand is only the part that grants life.
+ then you wouldn't be able to stay sane. What you understand is nothing more the part that grants life.
 }
 [sha:ddbc08f94e409ab5d792d0a6c674e85c1a39b74e]{
 -- Page 438, Offset 32928.
 -- 死が視えるのなら―――とても、立ってなんていられない」
-If you could truly see death―――you wouldn't be able to even stand."
+If you could truly see death―――you wouldn't even be able to stand."
 }
 [sha:0dcfa1045baef15addb3b33f3b7b1ba1aa189726]{
 -- Page 438, Offset 32929.
 -- 喩えるのなら、それは月世界。
-To illustrate, it would be like the Lunar World.
+You could liken it to the surface of the moon.
```

Comment:
woah... like the title... moon... so crazy...

Reply:
I think that this is the chapter title (Lunar World) but nobody _says_ that so I think people will just have to make the connection on their own like good little literate media consumers

Reply:
Lol, this wasn't actually made having looked at the old version of the line. I just think this line is very dumb and he uses the god damn moon analogy way too much